website/source/docs/auth/aws-iam.html.md

```diff
@@ -0,0 +1,606 @@
+---
+layout: "docs"
+page_title: "Auth Backend: AWS-IAM"
+sidebar_current: "docs-auth-aws-iam"
+description: |-
+  The aws-iam backend allows automated authentication AWS IAM principals.
+---
+
+# Auth Backend: aws-iam
+
+The aws-iam auth backend provides a secure introduction mechanism for AWS IAM
+principals. This allows you to reduce the problem of securely introducing a
+Vault token to the problem of securely introducing AWS IAM credentials, which
+AWS has already solved in a number of use cases, such as via EC2 Instance
+Profiles and IAM roles attached to Lambda functions. It also allows you to have
+a consistent workflow between developers working on a local laptop with tools
+such as Hologram.
+
+## Comparison with AWS-EC2 Authentication Backend
+
+The AWS-IAM and AWS-EC2 authentication backends serve similar purposes. Both
```

Comment:
Also IAM is cross region and used for other things than Vault authentication, and you may well want to give access to a shared IAM role but change or limit Vault policies based on other factors (of which VPC might be one, but only one)...which in theory could be far easier than duplicating and managing IAM roles. 

In both this and the AWS-EC2 case the unfortunate reality is that AWS could use API enhancements that would make both better (and ideally allow sensible convergence). Until then both approaches have pros and cons and will work better in different situations.